cmd/google_cloud_ops_agent_engine/main.go

```diff
@@ -31,6 +32,36 @@ var (
 	stateDir = flag.String("state", "/var/lib/google-cloud-ops-agent", "path to store agent state like buffers")
 )
 
+func runStartupChecks(service string) error {
+	// To run checks in each subagent service we could
```

Comment:
Do we want to implement this right now?
If not, we can either just remove this or create a bug and reference it here, so it doesn't get lost in the limbo.

Reply:
No, discussed with @jinghan-ma about this. Its documented in the design. It would be easy to add in the subagent units, but the problem is that Health Check results would show in the subagent services.